packages/flutter/lib/src/material/stepper.dart

```diff
@@ -232,6 +240,27 @@ class Stepper extends StatefulWidget {
   /// The index into [steps] of the current step whose content is displayed.
   final int currentStep;
 
+  /// give background color to header
+  final Color? backgroundHeaderColor;
+
+  /// set the BoderRadius to header
+  final BorderRadiusGeometry? headerRadius;
+
+  /// set the color to text of active step
+  final Color? activeCircleTextColor;
+
+  /// set the background color to step
+  final Color? circleColor;
```

Comment:
This variable seems to only impact a step with `isActive == true`, which is an important detail to capture in the docstring.

Reply:
I also wonder whether this should be paired with an `inactiveCircleColor` property 🤔